pygmt/tests/test_basemap.py

```diff
@@ -3,16 +3,16 @@
 """
 import pytest
 from pygmt import Figure
+from pygmt.exceptions import GMTInvalidInput
 
 
-@pytest.mark.mpl_image_compare
```

Comment:
The test is no longer valid because `Figure.basemap` no longer raises a `GMTInvalidInput` exception when these parameters are not given. I think we can remove this test.

Instead, we should add a test (similar to the first example in https://github.com/GenericMappingTools/pygmt/issues/2365) to `test_subplot.py` to make sure it always works.

Reply:
> The test is no longer valid because `Figure.basemap` no longer raises a `GMTInvalidInput` exception when these parameters are not given. I think we can remove this test.

This is done in  commit https://github.com/GenericMappingTools/pygmt/pull/2417/commits/37710f1e55973edd515451d04698599c9513509c.

> Instead, we should add a test (similar to the first example in #2365) to `test_subplot.py` to make sure it always works.

So far, I added code for a test (please see commit https://github.com/GenericMappingTools/pygmt/pull/2417/commits/0716819dc9015971fd08dd7c6d1596a72fc6f61c). Will look up in the Contributors Guide how testing of plots works 🙂.

Reply:
Let me know if you need any help with testing plots.

Reply:
Thanks @seisman!

@GenericMappingTools/pygmt-maintainers it seems like I (link to my DAGsHub account https://dagshub.com/yvonnefroehlich) have to be added to the PyGMT organization / project on DAGsHub as a collaborator.

Btw the link https://dagshub.com/GenericMappingTools/pygmt/settings/collaboration given in the Contributors Guide (https://www.pygmt.org/dev/contributing.html#using-data-version-control-dvc-to-manage-test-images) gives me a 404 error.

Reply:
> it seems like I (link to my DAGsHub account https://dagshub.com/yvonnefroehlich) have to be added to the PyGMT organization / project on DAGsHub as a collaborator.

Done.



> Btw the link https://dagshub.com/GenericMappingTools/pygmt/settings/collaboration given in the Contributors Guide (https://www.pygmt.org/dev/contributing.html#using-data-version-control-dvc-to-manage-test-images) gives me a 404 error.

I think this link can only be accessed by users who have write/admin permissions.

Reply:
I tried to follow the steps in the Contributors Guide. It seems as it has worked, and the test `DVC image diff / DVC image diff (pull_request)` passed. But not completely sure. Is there anything on DAGsHug, which reports whether everything is fine or not? So far, I can not see anything happen there.

Reply:
Yes, everything works as expected. All tests pass, and the dvc image diff workflow also works (see https://github.com/GenericMappingTools/pygmt/pull/2417#issuecomment-1468394918).